x-pack/test/detection_engine_api_integration/security_and_spaces/tests/update_rules.ts

```diff
@@ -175,11 +179,213 @@ export default ({ getService }: FtrProviderContext) => {
             },
           },
         ];
+        // This should be "no_actions"
         outputRule.throttle = '1m';
         const bodyToCompare = removeServerGeneratedPropertiesIncludingRuleId(body);
         expect(bodyToCompare).to.eql(outputRule);
       });
 
+      it('should update a single rule property and remove the action', async () => {
```

Comment:
This test is like the one above but doesn't use legacy actions. Notice how when we do an update of the name it removes the actions completely.